src/ArticleView.jsx

```diff
@@ -0,0 +1,34 @@
+
+import { useState, useEffect } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+import axios from "axios";
+import Button from "react-bootstrap/Card";
+
+const ArticleView = () => {
+    const { articleId } = useParams();
+    const [article, setArticle] = useState(null);
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        axios.get(`https://news-that-matters.onrender.com/api/articles/${articleId}`)
```

Comment:
As per my previous comments on the last PR, this logic should live in an api.js file 👍